srcpkgs/volta/template

```diff
@@ -11,11 +11,7 @@ maintainer="Alex Lohr <alex.lohr@logmein.com>"
 license="BSD-2-Clause"
 homepage="https://volta.sh/"
 distfiles="https://github.com/volta-cli/volta/archive/v${version}.tar.gz"
-checksum=df4668d4a5b187fd6a332f2099a5a90e14b5393da7de5636e638f6aee3d9fd29
-
-pre_build() {
```

Comment:
This is no longer needed, as volta switched from reqwest to attohttpc, which supports libressl out of the box.